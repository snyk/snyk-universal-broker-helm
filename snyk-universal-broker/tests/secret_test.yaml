values:
  - ../values.yaml
  - fixtures/default_values.yaml
suite: Secret
templates:
  - templates/secret.yaml
tests:
   - it: Renders a valid universal broker secret
     asserts:
      - matchSnapshot:
          path: stringData

   - it: should render broker platform auth secret with expected values
     asserts:
      - isKind:
            of: Secret
      - equal:
          path: stringData.deploymentId
          value: "8B338A3B-424A-497E-836E-5E0F9486605A"
      - equal:
          path: stringData.clientId
          value: "8B338A3B-424A-497E-836E-5E0F9486605A"
      - equal:
          path: stringData.clientSecret
          value: "8B338A3B-424A-497E-836E-5E0F9486605A"
      - equal:
          path: stringData.mygithubtoken
          value: "8B338A3B-424A-497E-836E-5E0F9486605A"

   - it: should render broker platform auth secret with expected values
     set:
        existingAuthSecret: "some-existing-secret"
     asserts:
      - hasDocuments:
          count: 0
