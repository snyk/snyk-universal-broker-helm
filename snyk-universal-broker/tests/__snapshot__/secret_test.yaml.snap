Renders a valid universal broker secret:
  1: |
    clientId: 8B338A3B-424A-497E-836E-5E0F9486605A
    clientSecret: 8B338A3B-424A-497E-836E-5E0F9486605A
    deploymentId: 8B338A3B-424A-497E-836E-5E0F9486605A
    mygithubtoken: 8B338A3B-424A-497E-836E-5E0F9486605A
